api/app/wildfire_one/schema_parsers.py

```diff
@@ -240,3 +241,49 @@ def parse_hourly_actual(station_code: int, hourly):
     # don't write the HourlyActual to our database if every value is invalid. If even one
     # weather variable observed is valid, write the HourlyActual to DB.
     return None if is_obs_invalid else hourly_actual
+
+
+async def weather_station_group_mapper(raw_station_groups_by_owner: Generator[dict, None, None]) -> List[WeatherStationGroup]:
+    """ Maps raw weather station groups to WeatherStationGroup"""
+    weather_station_groups = []
+    async for raw_group in raw_station_groups_by_owner:
+        weather_station_groups.append(WeatherStationGroup(
+            display_label=raw_group['displayLabel'],
+            group_description=raw_group['groupDescription'],
+            group_owner_user_guid=raw_group['groupOwnerUserGuid'],
+            group_owner_user_id=raw_group['groupOwnerUserId'],
+            id=raw_group['id']))
+
+    return weather_station_groups
+
+
+def weather_stations_mapper(stations) -> List[WeatherStationGroupMember]:
```

Comment:
Does `stations` have a specific type we could indicate? If not, no worries.
 Same question for `unique_weather_stations_mapper` below.

Reply:
At this point it's raw json from WF1 so unfortunately it doesn't.